crates/ink/tests/ui/event/fail/conficting_attributes.stderr

```diff
@@ -0,0 +1,13 @@
+error: Only a single `#[ink(anonymous)]` attribute allowed.
```

Comment:
Here again hitting a misleading error message

Reply:
What's misleading here? You can't have multiple nested `#[ink()]` attributes. it doesn't matter if they are the same or different